src/llvm-final-gc-lowering.cpp

```diff
@@ -214,6 +214,36 @@ bool FinalLowerGC::runOnFunction(Function &F)
     }
 
     return true;
+
+    verify_skip:
+#ifdef JL_VERIFY_PASSES
+    for (auto &BB : F) {
+        for (auto &I : make_early_inc_range(BB)) {
+            auto *CI = dyn_cast<CallInst>(&I);
+            if (!CI)
+                continue;
+
+        Value *callee = CI->getCalledOperand();
+        assert(callee);
+        auto IS_INTRINSIC = [&](auto intrinsic) {
+            auto intrinsic2 = getOrNull(intrinsic);
+            if (intrinsic2 == callee) {
+                errs() << "Final-GC-lowering didn't eliminate all intrinsics'" << F.getName() << "', dumping entire module!\n\n";
+                errs() << *F.getParent() << "\n";
+                abort();
+            }
+        };
+        IS_INTRINSIC(jl_intrinsics::newGCFrame);
+        IS_INTRINSIC(jl_intrinsics::pushGCFrame);
+        IS_INTRINSIC(jl_intrinsics::popGCFrame);
+        IS_INTRINSIC(jl_intrinsics::getGCFrameSlot);
+        IS_INTRINSIC(jl_intrinsics::GCAllocBytes);
+        IS_INTRINSIC(jl_intrinsics::queueGCRoot);
+        IS_INTRINSIC(jl_intrinsics::safepoint);
+        }
+    }
+#endif
+    return false;
```

Comment:
With something like #56188 this may fail if we use `llvm.compiler.used`.

Reply:
I'm a bit confused? Why would it fail? After this it should be an addrspacecast